src/sr_od/challenge_mission/use_trailblaze_power.py

```
@@ -90,8 +96,27 @@ def _get_current_challenge_times(self) -> int:
             return min(8, current_challenge_times)
-        elif self.mission.cate.cn in ['侵蚀隧洞', '培养目标']:
+        elif self.mission.cate.cn in ['侵蚀隧洞']:
             return min(6, current_challenge_times)
+        elif self.mission.cate.cn == '培养目标':
+            if self.mission_power_ocr == 0:
+                # 识别单次体力消耗
+                area1 = self.ctx.screen_loader.get_area('挑战副本', '预计消耗体力-其他')
+                ocr_result_list = self.ctx.ocr_service.get_ocr_result_list(self.last_screenshot, rect=area1.rect)
+                if len(ocr_result_list) > 0:
+                    power_unit_str = ocr_result_list[0].data
+                    if power_unit_str[0] in ['×', 'x']:
```

Ruff (0.15.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)